src/ViperIDE.html

```diff
@@ -14,7 +14,7 @@
         <div>
             <button onclick="app.toggleSideMenu()"><i class="fa-solid fa-bars"></i></button>
             <span class="logo">ViperIDE</span>
-            <button title="Save File [Ctrl+S]"   onclick="app.saveCurrentFile()" id="btn-save"><i class="fa-solid fa-download"></i></button>
+            <button title="Save File [Ctrl/Cmd+S]"   onclick="app.saveCurrentFile()" id="btn-save"><i class="fa-solid fa-download"></i></button>
```

Comment:
Please note that this is overridden during internationalization:

https://github.com/vshymanskyy/ViperIDE/blob/cf6909562db197498e59f430550e6ef4ecb7f36f/src/app.js#L818

It also presents a chance to detect the OS dynamically and replace with `Ctrl` or `Cmd` accordingly

Reply:
Ah, OK. I was wondering how that worked. This makes sense. I'll make some changes here including the dynamic detection.